docs/http-api.md

```diff
@@ -74,13 +98,19 @@ POST /render
 
 * header `X-Org-Id` required
 * maxDataPoints: int (default: 800)
-* target: mandatory. one or more metric names or patterns, like graphite.  
+* target: mandatory. one or more metric names or patterns, like graphite.
```

Comment:
in case you didn't know, double space at end of line is the markdown way of forcing a new line.   It's weird but that's how it is.  
(https://daringfireball.net/projects/markdown/syntax#p)
I like the following note to go on a new line.